Basics/Fibonacci Series.cpp

```diff
@@ -0,0 +1,63 @@
+#include<iostream>
+using namespace std;
+
+int fibonacciRecursive (int n)
+{
+		if ( n == 0 )
+         return 0;
+        else if ( n == 1 )
+         return 1;
+        else
+         {
+		 	return ( fibonacciRecursive(n-1) + fibonacciRecursive(n-2) );
+		 }
+
+
+
+}
+
+int fibonacciIterative(int n)
+{
+	int a , b , i , c ;
+	i = 2 ;
+	a = 0 ; 
+	b = 1 ;
+	if(n==1) //n=1 and n=2 are given cases
+	{
+		cout<<a;
```

Comment:
no need to cout . return that fibonacci number instead. int finbonacci should return the n'th fibonaci . if n=1 , return a , n==2 , return b else calculate that c and return it , dont output it here in function.